internal/kubernetes.go

```diff
@@ -45,14 +45,21 @@ func (exporter *Exporter) parseAllKubeSecrets() ([]*certificateRef, []error) {
 		for _, secret := range secrets {
 			for _, secretType := range exporter.KubeSecretTypes {
 				typeAndKey := strings.Split(secretType, ":")
-
-				if secret.Type == v1.SecretType(typeAndKey[0]) && len(secret.Data[typeAndKey[1]]) > 0 {
-					output = append(output, &certificateRef{
-						path:          fmt.Sprintf("k8s/%s/%s", namespace, secret.GetName()),
-						format:        certificateFormatKubeSecret,
-						kubeSecret:    secret,
-						kubeSecretKey: typeAndKey[1],
-					})
+				kubeSecretKeys, err := getMatchingKeys(secret.Data, typeAndKey[1])
+				if err != nil {
+					outputErrors = append(outputErrors, fmt.Errorf("failed to fetch keys for secret from namespace \"%s\": %s", namespace, err.Error()))
+					continue
+				}
+				for _, kubeSecretKey := range kubeSecretKeys {
+
```

Comment:
In a cosmetic standpoint, this empty line could be removed.

Reply:
sure